### PR TITLE
CCM-5394 Contact details API documentation

### PR DESCRIPTION
**Pull request by Ian-Hodges**
## Summary

Updating the API documentation for the new override contact details feature.

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Checklist
* [x] Brief description of work completed, and any technical decisions made as part of the PR
* [x] PR link added as a comment to the relevant JIRA ticket
* [x] PR link shared on Slack and/or Teams
* [x] 2 reviews received
* [x] Tester approval

## Changes

Bloomreach link: https://cms-training2.nhsd.io/cms/content/path/content/documents/corporate-website/developer/api-catalogue/nhs-notify/iaho3

### Overview

image

### Batch 400s

image

image

image

### Single 400s

image

image

image

### Batch schema/example

image

image

### Single schema/example

image

image

### CODE REVIEW

**Review by simonlabarere — APPROVED** [review]

File: specification/schemas/components/Recipient.yaml [hunk 1]
```diff
@@ -14,5 +14,36 @@ properties:
...
+      email:
+        type: string
+        pattern: '[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}'
+        minLength: 6
+        maxLength: 90
+        description: Overriding email address for recipient.
+        example: "recipient@nhs.net"
+      sms:
+        type: string
+        description: Overriding sms telephone number for recipient. Validated using the `libphonenumber-js/mobile` npm package.
```
[the remark this answers is not in the document]
Comment (aaron-downing) [reply specification/schemas/components/Recipient.yaml:30]: Maybe we just say valid UK phone number?

File: specification/documentation/APIDescription.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
 ## Overview
 
-Use this API to send messages to citizens via email, SMS, the NHS App or letter.
+Use this API to send messages to citizens via NHS App, email, text message or letter.
```
Comment (tdroza-nhs) [inline specification/documentation/APIDescription.md:3]: I wonder if we should put "text message (SMS)" to match how we list the channels on the Features page of the service catalogue? https://digital.nhs.uk/services/nhs-notify/features#:~:text=emails-,text%20messages%20(SMS),-letters

**Review by tdroza-nhs — APPROVED** [review]

File: specification/schemas/components/Recipient.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (aaron-downing) [reply specification/schemas/components/Recipient.yaml:30]: May be best to say valid UK phone number

[the remark this answers is not in the document]
Comment (Ian-Hodges) [reply specification/schemas/components/Recipient.yaml:30]: We can change the documentation if it changes?
I think it's good to let clients align with our exact validation.

[the remark this answers is not in the document]
Comment (aaron-downing) [reply specification/schemas/components/Recipient.yaml:30]: ok, may be worth still adding that it's GB/UK phone numbers

File: specification/documentation/APIDescription.md (hunk 2 above)
Comment (Ian-Hodges) [reply specification/documentation/APIDescription.md:3]: As discussed with Jake, leaving as-is.

File: specification/schemas/components/Recipient.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ian-Hodges) [reply specification/schemas/components/Recipient.yaml]: Good point. Updated.

**Review by danh-nhs — APPROVED** [review]

**Pull request merged**